tools/common_lib/src/gemm.h

```diff
@@ -1327,6 +1327,49 @@ class GemmUmdD3d12Dispatcher : public GemmBaseDispatcher
         gemm_.execute(stream, args);
     }
 
+
+    ConformanceResult validate_conformance(ID3D12CommandQueue* command_queue,
+            ID3D12CommandAllocator* command_allocator, ID3D12GraphicsCommandList* command_list, bool print_mismatche) override
+    {
+        auto dump_buffer_to_file = [&](const auto& buffer, const auto& file_name)
```

Comment:
Similar lambda is in convolution umd dispatcher - could it be moved to be separate function?